workshops/templates/workshops/all_events.html

```diff
@@ -45,9 +45,9 @@
             </a>
           {% endif %}
         </td>
-        <td {% if not event.get_repository_url %}class="warning"{% endif %}>
-          {% if event.get_repository_url %}
-          <a href="{{ event.get_repository_url }}">{{ event.get_repository_url|cut:"https://github.com/" }}
+        <td {% if not event.repository_url %}class="warning"{% endif %}>
+          {% if event.repository_url %}
+          <a href="{{ event.repository_url }}">{{ event.repository_url|cut:"https://github.com/" }}
```

Comment:
I'm against cutting the URL in this place anyway, but I believe it was historical code so I left it intact. Since you're against then I'll remove it.